src/pyk/kcfg/kcfg.py

```diff
@@ -958,22 +1016,28 @@ def lift_splits(self) -> bool:
             -   bool: An indicator of whether or not at least one split lift was performed.
         """
 
-        result = False
-        while True:
-            splits_to_lift = [
-                node.id
-                for node in self.nodes
-                if self.splits(source_id=node.id) != [] and self.edges(target_id=node.id) != []
-            ]
-            if len(splits_to_lift) == 0:
-                break
-            for node_id in splits_to_lift:
-                try:
-                    self.lift_split(node_id)
-                    result = True
-                except AssertionError as err:
-                    _LOGGER.warning(str(err))
-        return result
+        def lift_split(finder: Callable, lifter: Callable) -> bool:
+            while True:
+                result = False
+                splits_to_lift = [
+                    node.id
+                    for node in self.nodes
+                    if self.splits(source_id=node.id) != [] and finder(target_id=node.id) != []
+                ]
+                for node_id in splits_to_lift:
+                    try:
+                        lifter(node_id)
+                        result = True
+                    except AssertionError as err:
+                        _LOGGER.warning(str(err))
+                if not result or len(splits_to_lift) == 0:
+                    break
+            return result
+
+        def fold_lift(result: bool, finder_lifter: tuple[Callable, Callable]) -> bool:
+            return lift_split(finder_lifter[0], finder_lifter[1]) or result
```

Comment:
```suggestion
        def _lift_split(finder: Callable, lifter: Callable) -> bool:
            while True:
                result = False
                splits_to_lift = [
                    node.id
                    for node in self.nodes
                    if self.splits(source_id=node.id) != [] and finder(target_id=node.id) != []
                ]
                for node_id in splits_to_lift:
                    try:
                        lifter(node_id)
                        result = True
                    except AssertionError as err:
                        _LOGGER.warning(str(err))
                if not result or len(splits_to_lift) == 0:
                    break
            return result

        def _fold_lift(result: bool, finder_lifter: tuple[Callable, Callable]) -> bool:
            return lift_split(finder_lifter[0], finder_lifter[1]) or result
```

Prefix internal functions with `_`.

Reply:
Done.